src/applications/mhv/medical-records/api/MrApi.js

```diff
@@ -212,26 +212,39 @@ export const getVaccine = id => {
 };
 
 /**
- * Get a pdf of a single vaccine
- * @param {Long} folderId
+ * Get a pdf of a list of all allergies
  * @returns json with base64 of a pdf
  */
-export const getVaccinePdf = id => {
```

Comment:
It looks like this is still being used in [AllergyDetails](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/mhv/medical-records/containers/AllergyDetails.jsx#L10C1-L10C1)

Reply:
Oh! Thanks, had fixed this locally but was saving that file for another PR. I'll add the necessary lines to this PR.